README.md

```diff
@@ -86,14 +86,14 @@ The version number may have trailing pre-release version identifiers to indicate
 - Build metadata may be denoted by a plus sign followed by 4 or 5 digits, such as  `1.2.3-preview+5678` or `1.2.3+5678`. Build metadata does not figure into the precedence order.
 
 ## Future Plans
-- Expand SQL Server 2016 feature support (example: Always Encrypted)
+- Expand SQL Server 2016 feature support (example: Azure AD)
```

Comment:
Let's spell out acronyms fully

Reply:
This pull request has been merged, @v-mabarw 
If we are going to make changes, we will create another pull request. Please let us know.